scripts/run-dependencies.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+here=$(dirname $0)
+
+echo "using orderly path:"
+echo $MONTAGU_ORDERLY_PATH
+
+config_path=$(realpath $here/../config)
+export MONTAGU_API_VERSION=$(<$config_path/api_version)
+export MONTAGU_DB_VERSION=$(<$config_path/db_version)
+export MONTAGU_ORDERLY_SERVER_VERSION=$(<$config_path/orderly_server_version)
+
+COMPOSE_FILE=$here/../scripts/docker-compose.yml
+
+docker-compose -f $COMPOSE_FILE pull
+docker-compose -f $COMPOSE_FILE --project-name montagu up -d
+
+function cleanup() {
```

Comment:
I don't know much about the scope of bash functions. Does it matter that you've got two functions called cleanup, one declared in a script that's invoked from the other?

Reply:
I don't think so, by default this one won't be callable from the other or vice versa.